pkg/aws/aws.go

```diff
@@ -38,6 +38,13 @@ const (
 	// It is optional.
 	ConfigRole = "role"
 
+	// ConfigEFSVaultName represents the key for the EFS vault name in the map
+	// "config". It is optional.
+	ConfigEFSVaultName = "efsVaultName"
+	// EfsBackupVaultNameEnv describes the environment variable used to set the
+	// backup vault name.
+	EfsBackupVaultNameEnv = "EFS_BACKUP_VAULT_NAME"
```

Comment:
Remove this

Reply:
Why? That's how it will be referenced through K10?